doc/changelog.rst

```diff
@@ -33,6 +34,7 @@ Localization
 Admin
 ~~~~~
 
+- Enable all methods for order creator
```

Comment:
What methods? "Methods" like member functions of a class? (That's what I think first as a programmer.) Methods of communication? Methods of creating order? Methods of entering customer details? There is so many "methods"... :)

How about "Enable all shipping and payment methods in order creator" ?